.storybook/config.js

```diff
@@ -0,0 +1,20 @@
+import { configure } from '@storybook/react';
+
+// Our Application's CSS.
+import '../src/index.css';
+
+// Import WordPress admin-side styles.
+import '../../../wordpress/wp-admin/css/admin-menu.css';
```

Comment:
We may need a more solid way to do this. As long as we can control where this code lives and where WP lives it's ok but might need to become configurable.